packages/sdk-components-react/src/__generated__/vimeo.props.ts

```diff
@@ -19,7 +17,6 @@ export const props: Record<string, PropMeta> = {
   accessKey: { required: false, control: "text", type: "string" },
   autoFocus: { required: false, control: "boolean", type: "boolean" },
   className: { required: false, control: "text", type: "string" },
-  contentEditable: { required: false, control: "text", type: "string" },
```

Comment:
not really necessary but isn't a complex type, so I think this indicates mistake in the logic

Reply:
It's "Booleanish" type

Reply:
can we detect such types from the data?

Reply:
The question is which type we should prefer
```
boolean | 'true' | 'false'
```

Reply:
form that choice this is clearly a boolean

Reply:
Here yes but
```
'aria-checked'?: boolean | 'false' | 'mixed' | 'true' | undefined;
'aria-current'?: boolean | 'false' | 'true' | 'page' | 'step' | 'location' | 'date' | 'time' | undefined;
```